apps/CA.pl.in

```diff
@@ -132,9 +132,7 @@ if ($WHAT eq '-newcert' ) {
     $RET = run("$REQ -new -keyout $NEWKEY -out $NEWREQ $DAYS $EXTRA{req}");
     print "Request is in $NEWREQ, private key is in $NEWKEY\n" if $RET == 0;
 } elsif ($WHAT eq '-newreq-nodes' ) {
-    # create a certificate request
-    $RET = run("$REQ -new -nodes -keyout $NEWKEY -out $NEWREQ $DAYS $EXTRA{req}");
-    print "Request is in $NEWREQ, private key is in $NEWKEY\n" if $RET == 0;
+    die "Flag removed; please use -extra-req-nodes instead.\n";
```

Comment:
Are you sure that it's not `-extra-req -nodes`? And the message is confusing, because it gives you impression that it *replaces* -newreq-nodes, but it doesn't. It looks like alternative is `-newreq -extra-req -nodes`.

But what exactly is the license-able claim? Existence of the flag or its implementation? Because if latter, then one can as well do something like
```
} elsif ($WHAT =~ '/\-newreq(\-nodes)?/' ) {
    # create a certificate request
    $RET = run("$REQ -new $1 -keyout $NEWKEY -out $NEWREQ $DAYS $EXTRA{req}");
    print "Request is in $NEWREQ, private key is in $NEWKEY\n" if $RET == 0;
```